src/webapp/components/table/statistic-table/MedianRow.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import _ from "../../../../domain/entities/generic/Collection";
+import { TableCell, TableRow } from "@material-ui/core";
+import styled from "styled-components";
+import { StatisticTableProps } from "./StatisticTable";
+
+export type TableColumn = {
+    value: string;
+    label: string;
+    dark?: boolean;
+};
+
+type MedianRowProps = {
+    columns: TableColumn[];
+    rows: {
+        [key: TableColumn["value"]]: string;
+    }[];
+    calculateColumns: TableColumn["value"][];
+};
+
+export const MedianRow: React.FC<MedianRowProps> = React.memo(
+    ({ rows, columns, calculateColumns }) => {
+        const calculateMedian = (
```

Comment:
I think that this function together with the function in `PercentTargetMetRow` (`calculatePercentTargetMet`) should be also in a custom hook. For me, the component seems to have too much logic and that should be in a custom hook, the component would only be limited to display information :thinking:. Both components are very similar, would be possible to have only one to show the calculations and pass as props the function to calculate and other needed props?

Reply:
I agree I think I can make theses row more generic 👍